src/components/permissions.ts

```diff
@@ -62,9 +62,9 @@ const getUserPermissions = async (username: string) => {
 const buildTemplatePermissions = (templatePermissionRows: Array<PermissionRow>) => {
   const templatePermissions: TemplatePermissions = {} // TODO add type
 
-  templatePermissionRows.forEach(({ permissionType, templateName }: any) => {
-    if (!templatePermissions[templateName]) templatePermissions[templateName] = []
-    templatePermissions[templateName].push(permissionType)
+  templatePermissionRows.forEach(({ permissionType, templateCode }: PermissionRow) => {
```

Comment:
This is throwing an error because the interface `PermissionRow` up the top of the file is still defining `templateName` rather than `templateCode`

Reply:
ea27fa6e4617026b12af84f124f603451824da09

Oops sorry, I shouldn't be pushing commits without compiling